Product/Production/Services/CORE_X12DocumentSubmissionCore/pom.xml

```diff
@@ -51,6 +51,27 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <!-- Testing -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jmock</groupId>
```

Comment:
Those are the common libraries used across all the modules and these are only used for test, not packaged within the ear.